doctr/models/predictor/pytorch.py

```diff
@@ -57,6 +64,13 @@ def forward(
         if any(page.ndim != 3 for page in pages):
             raise ValueError("incorrect input shape: all pages are expected to be multi-channel 2D images.")
 
+        origin_page_shapes = [page.shape[:2] for page in pages]
```

Comment:
If this is a PyTorch tensor and not a numpy one, the axis order will be C, H, W so I think this won't work in that case

Reply:
Could you please help me here ? I don't know how to debug this. I used the following snippet to force using PyTorch but it seems that the channel is still last. When do we have PyTorch tensors ?
 
```python
from pathlib import Path

import matplotlib.pyplot as plt

from doctr.io import DocumentFile
from doctr.models.detection.zoo import detection_predictor
from doctr.models.predictor.pytorch import OCRPredictor
from doctr.models.recognition.zoo import recognition_predictor
from doctr.utils.visualization import visualize_page

file = Path(r"myfile.jpg")

doc = DocumentFile.from_images(file)
det_bs=2
reco_bs=128

det_predictor = detection_predictor('db_resnet50', pretrained=True, batch_size=det_bs, assume_straight_pages=True)
reco_predictor = recognition_predictor('crnn_vgg16_bn', pretrained=True, batch_size=reco_bs)

predictor = OCRPredictor(det_predictor=det_predictor, reco_predictor=reco_predictor, assume_straight_pages=True, straighten_pages=True)

# Plot OCR output
out = predictor(doc)

fig = visualize_page(out.pages[0].export(), doc[0], interactive=False)
plt.show()

```